vms/proposervm/vm.go

```diff
@@ -220,6 +227,34 @@ func (vm *VM) Initialize(
 	default:
 		return err
 	}
+
+	vm.proposerBuildSlotGauge = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "block_building_slot",
+			Help: "the post-durango slot in which the block was built",
+		})
+
+	if err = registerer.Register(vm.proposerBuildSlotGauge); err != nil {
+		return err
+	}
+
+	vm.acceptedBlocksSlotHistogram = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Name: "accepted_blocks_slot_histogram",
```

Comment:
I think the type of the metric already implies that it is a histogram.
```suggestion
			Name: "accepted_blocks_slot",
```